apps/sailfish/qml/components/PlayerControls.qml

```diff
@@ -76,6 +108,11 @@ Row {
         icon.height: 75; icon.width: 75
         height: 75; width: 75
         enabled: !!player
-        onClicked: playerControls.player.skipNext()
+        onClicked: {
+            if (settings.hapticsEnabled) {
+                rumbleEffect.start(2);
+            }
+            playerControls.player.skipNext()
+        }
```

Comment:
Robert, you are the Sailfish master so you decide, but wouldn't it make more sense to create a KodimoteButton.qml with all those identical things and just use that everywhere?